sass/_Pager.scss

```diff
@@ -1,32 +1,52 @@
 .coveo-pager-list {
   padding-left:0;
+  margin:0;
+  display: inline-block;
 }
 
 .coveo-pager-list-item {
-  border: 0;
+  @include defaultRoundedBorder();
   margin: 5px 6px 5px 6px;
-  padding: 0;
+  padding: 4px 8px;
   list-style: none;
   display: inline-block;
   vertical-align: baseline;
-  color: #bcc3ca;
-  font-size: $font-size-biggest;
-  &.coveo-active {
-    font-size: $font-size-huge;
-    color: $color-blueish-gray;
-  }
-  &:hover {
-    color: $color-blueish-gray;
+  color: $color-vibrant-blue;
+  font-size: $font-size-smaller;
+  &.coveo-active, &:hover {
+    color: $color-greyish-cyan;
+    background-color: $color-greyish-teal-blue;
   }
   cursor: pointer;
 }
 
+.coveo-pager-next{
```

Comment:
Couldn't you set the properties once for both `.coveo-pager-next` and `.coveo-pager-previous` since they have exactly the same properties ?